components/treelist/columns/menu.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Column Menu
+page_title: TreeList - Column Menu
+description: Use the Column Menu for the TreeList to show a menu that allows you to perform high-level column customization.
```

Comment:
What are "high-level customizations" (which ones are "low-level"?) and why do we need to label them if we list them afterwards? And if we don't list them, like here, maybe we should.

In other words, don't introduce unnecessary terminology.